tsconfig.json

```diff
@@ -15,6 +15,6 @@
     "jsx": "preserve",
     "incremental": true
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/components/table/DataTable.jsx"],
```

Comment:
`jsx` files React without Typescript. Change the `DataTable.jsx` to `DataTable.tsx` file and remove this config